indicators/ics.yml

```diff
@@ -0,0 +1,20 @@
+title: International Card Services phishing kit
+description: |
+    Detects a phishing kit for a creditcard processor which uses the same hidden value across various domains.
+    
+references:
+- https://urlscan.io/result/5cea2922-3642-4dc0-b41b-89e914a70f94
+- https://urlscan.io/result/44382989-2570-4b17-9fe8-962b6e341ddb
+- https://urlscan.io/result/7d2a26cd-8307-46be-9991-e691c962c2ea
+- https://urlscan.io/result/52ccdfe2-5af4-4b5b-a70c-73e2cd7b56e6
+
+detection:
+
+    did_proxy:
+      html|contains: <input id="did_proxy" name="did_proxy" type="hidden" value="1:zEUeQFVqXRrb1FthfkZ64J1LHpob1ksgZd7jHNpUxXxlq0gz2-i7oZP9U70asvhwgYSKXzVQArfJATAYu8N_bw">
```

Comment:
I couldn't actually find this value in the scans you referenced.

Should this be
```
<input id="did_proxy" name="did_proxy" type="hidden" value="1:zEUeQFVqXRrb1FthfkZ64ABdt_5GKQC5gxwQD8dmYXkPJuYmhQBLey8Rpq4xyTEU0FS3yivFtzoWrZDVqZEzXw">
```
instead?

Reply:
Any update on this rule? @jonade

Reply:
> I couldn't actually find this value in the scans you referenced.
> 
> Should this be
> 
> ```
> <input id="did_proxy" name="did_proxy" type="hidden" value="1:zEUeQFVqXRrb1FthfkZ64ABdt_5GKQC5gxwQD8dmYXkPJuYmhQBLey8Rpq4xyTEU0FS3yivFtzoWrZDVqZEzXw">
> ```
> 
> instead?

All the scans referenced within the rule itself and in this PR contain the HTML element defined in the rule, so I am not sure as to why you are getting a different value 🤨